Assets/Scripts/RewardedAdHandler.cs

```diff
@@ -14,6 +15,8 @@ public class RewardedAdHandler : MonoBehaviour, IUnityAdsLoadListener, IUnityAds
     [SerializeField] string _androidAdUnitId = "Rewarded_Android";
     [SerializeField] string _iOSAdUnitId = "Rewarded_iOS";
 
+    [Header("Implementation")]
+    [SerializeField] private UnityEvent OnReward = new UnityEvent();
     private bool debugMode;
```

Comment:
```suggestion
    [SerializeField] private UnityEvent _onReward = new UnityEvent();
    private bool _debugMode;
```

Reply:
refactored and checked the private variables in code